docs/tutorials/configure-external-signer-tls.md

```diff
@@ -145,22 +145,16 @@ web3signer --key-store-path=/Users/me/keyFiles/ \
 --tls-keystore-file=/Users/me/certs/web3signer_keystore.p12 \
 --tls-keystore-password-file=/Users/me/certs/web3signer_keystore_password.txt \
 --tls-known-clients-file=/Users/me/certs/knownClients.txt \
-eth2 --slashing-protection-enabled=false
 ```
 
 ## 5. Start Teku
 
 Start Teku and specify the [keystore](#teku-keystore-and-password-file) and [truststore](#2-create-the-truststore-and-password-file) created earlier, with the accompanying password files.
 
-:::caution
-
-This example connects to an [Infura] ETH1 endpoint, if running your own client, like [Hyperledger Besu], then add the JSON-RPC URL of the ETH1 node to the [`--eth1-endpoint`](../reference/cli/index.md#eth1-endpoint-eth1-endpoints) command line option.
-
-:::
+A [Holesky node] is the result of an execution layer client and a consensus layer client working together.
 
 ```bash
-teku --network=goerli \
---eth1-endpoint=https://goerli.infura.iov3/d0f36ceb0c324eef9984422efbc51f21 \
```

Comment:
Did you confirm with the devs that you don't need to replace `--eth1-endpoint` with `--ee-endpoint`? Because I'm pretty sure you still need that CLI option to connect to the execution client's (Besu's) RPC URL endpoint. You can see how it's used in our start Teku instructions: https://docs.teku.consensys.io/get-started/connect/testnet#beacon-node-and-validator-client

Their comments on this PR are just saying that you can't use the Infura URL endpoint for execution layer access.